lib/client/native_client.dart

```diff
@@ -374,6 +383,8 @@ class NativeClient {
       baseName = "lib$libraryName.dylib";
     } else if (Platform.isWindows) {
       baseName = "$libraryName.dll";
+    } else if (Platform.isIOS) {
```

Comment:
This should not be necessary because the static library will not be visible in runtime. Instead it is statically linked to the iOS app.

Reply:
Updated, seems working.